Directory.Build.props

```diff
@@ -124,4 +124,14 @@
   <PropertyGroup>
     <CustomAfterMicrosoftCommonTargets>$(RepoRoot)build\AfterCommonTargets.targets</CustomAfterMicrosoftCommonTargets>
   </PropertyGroup>
+  
+  <!-- API Compat -->
+  <PropertyGroup>
+    <!-- only validate that the current implementation is compatible
+         with the old one, not vice-versa (since latest may have new
+         API missing from old) -->
+    <RunApiCompatForSrc>true</RunApiCompatForSrc>
+    <RunMatchingRefApiCompat>false</RunMatchingRefApiCompat>
+    <MicrosoftDotNetApiCompatPackageVersion>1.0.0-beta.19225.5</MicrosoftDotNetApiCompatPackageVersion>
```

Comment:
This property should be moved to https://github.com/dotnet/machinelearning/blob/master/build/Dependencies.props